pyg_lib/csrc/sampler/cpu/neighbor_kernel.cpp

```diff
@@ -437,11 +495,16 @@ sample(const std::vector<node_type>& node_types,
         std::tie(begin, end) = slice_dict.at(src);
 
         if (!time_dict.has_value() || !time_dict.value().contains(dst)) {
+          if (!disjoint && return_sampled_info)
```

Comment:
Why do we need to gate this behind `disjoint`?

Reply:
Please look to the main reply.